src/controller/CommissioningDelegate.h

```diff
@@ -679,6 +679,16 @@ struct GeneralCommissioningInfo
     ;
 };
 
+struct ICDManagementClusterInfo
+{
+    bool isLIT                  = false;
+    bool checkInProtocolSupport = false;
+
+    BitMask<app::Clusters::IcdManagement::UserActiveModeTriggerBitmap> userActiveModeTriggerHint =
+        BitMask<app::Clusters::IcdManagement::UserActiveModeTriggerBitmap>();
+    CharSpan userActiveModeTriggerInstruction = CharSpan();
```

Comment:
This still needs the documentation I asked for.  I know https://github.com/project-chip/connectedhomeip/issues/31114 got filed, but given the history of not fixing such followups and the fact that it was not even labeled in a way that would make it trackable...  I don't believe it would get fixed, sorry.  Please fix this in this PR.

Reply:
agree, updated, thanks